pkg/cost/prometheus.go

```
@@ -0,0 +1,280 @@
+	}
+
+	var jobs []RawJobMetrics
+	for pod, labels := range labelsByPod {
+		cpu, hasCPU := cpuByPod[pod]
+		mem, hasMem := memByPod[pod]
+		if !hasCPU && !hasMem {
+			continue
+		}
+
+		jobs = append(jobs, RawJobMetrics{
+			Pod:     pod,
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(goimports)